### PR TITLE
Changes to support automated deployments

### PR DESCRIPTION
**Pull request by andrebriggs**
Example of automated deployment of this branch here: https://dev.azure.com/epicstuff/active%20learning/_releaseProgress?_a=release-environment-logs&releaseId=72&environmentId=73

### CODE REVIEW

File: devops/deployment/Deploy-Infrastructure.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -x
+set +x
```
Comment (jmspring) [inline devops/deployment/Deploy-Infrastructure.sh]: neither needed unless for debugging

File: devops/deployment/Deploy-Infrastructure.sh [hunk 2]
```diff
@@ -21,33 +21,53 @@ fi
 
 # Check if any of the args are empty
 if [ -z "$1" ]; then
-    echo "Usage: 'sh $0 (Configuration file)'"
-    exit 1
+    echo "Usage: 'sh $0 (Configuration file)' or SET Environment Variables"
```
Comment (jmspring) [inline devops/deployment/Deploy-Infrastructure.sh:23]: document environment variables?

Comment (andrebriggs) [reply devops/deployment/Deploy-Infrastructure.sh:23]: Addressed

File: devops/deployment/Deploy-Infrastructure.sh (hunk 1 above)
Comment (andrebriggs) [reply devops/deployment/Deploy-Infrastructure.sh]: Addressed

**Review by jmspring — APPROVED** [review]

**Pull request merged**